corehq/apps/app_manager/static/app_manager/js/section_changer.js

```diff
@@ -110,7 +110,20 @@ $(document).on("click", ".section-changer ul a", function (e) {
     e.preventDefault();
 });
 
+var expandSection = function (page, sectionSlug) {
+    var key = getKey(page, sectionSlug);
+    var $sectionChanger = $(".section-changer[data-collapse-key='" + page + "']");
+    var $form = $sectionChanger.closest("form");
+    var $panel = $form.find(".panel-appmanager[data-slug='" + sectionSlug + "']");
```

Comment:
could we add a non-style related class to the elements (makes migrations a lot easier with bootstrap)
for instance, `.js-appmanager-section-header`, or if all section slugs are unique, could we use an id instead? (also prefixed with `js-`

Reply:
(Dropped the section changer in favor of expandable/collapsible sections.)